src/assets/yii2-dynamic-form.js

```diff
@@ -115,7 +115,7 @@
         var count = _count($elem, widgetOptions);
 
         if (count < widgetOptions.limit) {
-            $toclone = widgetOptions.template;
+            $toclone = $(widgetOptions.template);
```

Comment:
why do we need to convert this into jQuery object? the original code would have expected the Browser DOM but we have now converted it into jQuery DOM. Will the existing functionality continue to work?